Storage/tests/Unit/StorageObjectTest.php

```diff
@@ -444,22 +437,16 @@ public function testDownloadsToFile()
         $bucket = 'bucket';
         $object = self::OBJECT;
         $stream = Utils::streamFor($string = 'abcdefg');
-        $this->connection->downloadObject([
-                'bucket' => $bucket,
-                'object' => $object,
-                'restOptions' => [
-                    'headers' => [
-                        'x-goog-encryption-algorithm' => 'AES256',
-                        'x-goog-encryption-key' => $key,
-                        'x-goog-encryption-key-sha256' => $hash,
```

Comment:
Do we need to drop these expectations?

Reply:
The reason I moved the expectations to a less specific version is because we now have callbacks intrinsically passed. So, I changes the expectations to:
```
Argument::allOf(
     Argument::withKey('bucket'),
     Argument::withKey('object'),
     Argument::withKey('restOptions')
)
```

Reply:
It seems like there is still some value in making sure the encryption headers are set properly, if it is possible to at least retain those.

Reply:
Done.